tests/core/test_storage.py

```diff
@@ -1209,6 +1209,15 @@ def test_number_of_fetched_records_can_be_limited_in_settings(self):
         results, count = limited.get_all(**self.storage_kw)
         self.assertEqual(len(results), 2)
 
+    def test_records_filtered_when_searched_by_string_field(self):
```

Comment:
Please add the test in the global StorageTest rather than the specific PostgreSQL one. All implemented backend should support this feature.